stimuli_ci_generation/src/server/main.py

```diff
@@ -63,7 +69,13 @@ def index():
         else:
             try:
                 generate_stimuli(participant_id, file_name)
-                return ("Generating stimuli images...", 202)
+                pub_msg(
+                    "Completed",
+                    SIE_IMG_PROCESSING_RESULT,
+                    participant_id,
+                    experiment_id,
+                )
+                return ("Stimuli generated", 202)
             except Exception as e:
                 print(e)
                 return ("Failed to generate stimuli", 204)
```

Comment:
Just a simple question here, why are you using 204 for a fail message? Shouldn't it be 400s or 500s?

Reply:
yep you are right! I will change it to a more appropriate one

Reply:
Pubsub doesn't play well with 400/500￼￼￼￼, it will leave message unacked and keep redeliver

Reply:
ahh that makes sense. I will do some research and add documentation there